packages/ui/src/ui/pages/groups/GroupEditorDialog/GroupEditorDialog.tsx

```diff
@@ -107,41 +122,58 @@ class GroupEditorDialog extends React.Component<GroupsPageTableProps> {
                 fields={[
                     {
                         type: 'tab-vertical',
-                        name: 'details',
-                        title: 'Details',
+                        name: 'general',
+                        title: 'General',
                         fields: [
                             {
-                                name: 'idm',
-                                type: 'plain',
-                                caption: 'Idm managed',
-                            },
-                            {
-                                name: 'size',
-                                type: 'plain',
-                                caption: 'Size',
-                            },
-                        ],
-                    },
-                    {
-                        type: 'tab-vertical',
-                        name: 'responsibles',
-                        title: 'Responsibles',
-                        fields: [
-                            {
-                                name: 'responsibles',
-                                type: 'acl-roles',
-                                caption: 'Responsibles',
-                                extras: {
-                                    placeholder: 'Enter login or name',
-                                },
-                            },
-                            {
-                                name: 'responsiblesComment',
-                                type: 'textarea',
-                                caption: 'Comment for IDM',
+                                name: 'groupName',
+                                type: 'text',
```

Comment:
We need a way to disable editing of the field, please add `AclApi.groups.allowRename` and check it here:
```ts
type: AclApi.groups.allowRename ? 'text' : 'plain',
```